src/dswx_sar/mosaic_rtc_burst.py

```diff
@@ -10,26 +10,156 @@
 import h5py
 from osgeo import osr, gdal
 import mimetypes
+import tempfile
+from scipy import ndimage
 
 from dswx_sar.dswx_runconfig import _get_parser, RunConfig
 from dswx_sar import dswx_sar_util
 
-logger = logging.getLogger('dswx_S1')
+logger = logging.getLogger('dswx_s1')
 
-def check_reprojection(geogrid_mosaic,
-                       rtc_image: str,
-                       nlooks_image: str = None) -> bool:
+
+def majority_element(num_list):
```

Comment:
Is this function to find the most frequent element in `num_list`? That being case. how about using `Counter` class in `collections`? Below is an example code:

```python
from collections import Counter

counter = Counter(num_list)
most_common = counter.most_common()
most_freq_elem = most_common[0][0]
```